### ISSUE TITLE
Trouble importing 'maxmind/lib/reader'

### ISSUE THREAD

**Issue by omar84** [runk/node-maxmind#176]
Hi @runk ,

the latest major update is not working for me.

for some weird reason, the project doesn't seem to recognize 'maxmind/lib/reader' as a module when I try to access Reader directly. I mean, typescript doesn't complain, but when running the project, node.js can't seem to find the imported path.

but I solved it when I added this line manually to index.ts in maxmind's folder in node modules.

`export { default as Reader } from './reader';
`

which, if compiled, would generate these lines in index.js:

```
var reader_2 = require("./reader");
exports.Reader = reader_2.default;
```

this will make `Reader` available in the main 'maxmind' package. and solve any weird sub-directory import issues. 

so that this will be possible:

```
import { Reader, CityResponse } from 'maxmind';

const buffer = await downloadFromCloudToStream();
const lookup = new Reader<CityResponse>(buffer);
let city = lookup.get('8.8.8.8');
```

can you please add this? I really want to be able to use the newest version. thanks

**Comment by runk** [comment]
Solved in v3.0.3

**Comment by omar84** [comment]
@runk @kevcenteno  thank you thank you thank you :)
